dart/dynamics/Skeleton.cpp

```diff
@@ -7917,7 +7917,7 @@ std::map<std::string, Eigen::Vector3s> Skeleton::getGyroMapReadings(
   for (auto& pair : gyros)
   {
     Eigen::Vector3s gyroInBodyFrame = pair.second.first->getAngularVelocity(
-        pair.second.first, pair.second.first);
+        Frame::World(), pair.second.first);

```

Comment:
Should this variable now be `gyroInWorldFrame`? Not sure if the docs for this method should be updated too.

Reply:
This was actually a super dumb bug. The first argument (which I always forget about) is the frame you're measuring the angular velocity _with respect to_, and the second argument is the coordinate frame to return results in. Before this change, this was measuring angular velocity with respect to itself, which always returned 0. The fix is to measure angular velocity with respect to the World instead, but still report results in the local body frame. Now the function actually does what it's supposed to.